JSON需要更新

用户登录中
(node:[17](https://github.com/Dong-Guan-Saohuang-Dadui/test/runs/6432043573?check_suite_focus=true#step:6:17)69) Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.
(Use `node --trace-warnings ...` to show where the warning was created)
Sleep for 3082ms...
正在获取前一天的疫情填报信息
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at getDailyReportFormData (file:///home/runner/work/test/test/src/index.ts:69:43)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async file:///home/runner/work/test/test/src/index.ts:143:[22](https://github.com/Dong-Guan-Saohuang-Dadui/test/runs/6432043573?check_suite_focus=true#step:6:22)
Error: Unexpected end of JSON input
 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.

--- a/src/index.ts
+++ b/src/index.ts
@@ -17,6 +17,9 @@ async function login(
     const cookieJar = new CookieJar();
     const client = got.extend({
         cookieJar,
+        headers: {
+            "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36"
+        },
         retry: {
             limit: RETRY,
             methods: ["GET", "POST"],

